src/Map/map.js

```diff
@@ -60,14 +57,14 @@ export const getMap = ({
       }}
     >
       {filteredMarkers &&
-        filteredMarkers.map((marker, index) => (
+        filteredMarkers.map(marker => (
           <Marker
             coordinate={{
               latitude: marker && marker.lat,
               longitude: marker && marker.lng,
             }}
             style={{ alignItems: 'center', justifyContent: 'center' }}
-            key={`marker ${index}`}
+            key={`${marker.lat}-${marker.lng}`}
```

Comment:
Do we have something preventing 2 markers with the same latitude and longitude from being rendered?